### ISSUE TITLE
[1.38.0-beta.1]  Dismiss issue text area is unlabelled and missing helper text

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1468]
This has an orphened label and no name for screen reader users. The helper text is not read.

<img width="1131" height="480" alt="Image" src="https://github.com/user-attachments/assets/34d2525c-4685-4d96-a568-6f348e942083" />

```
<div class="edac-rich-textarea-wrapper">
   <label class="edac-rich-textarea-label" style="">Comment (optional)</label>
   <div class="edac-rich-textarea-toolbar">
       <button type="button" class="components-button is-small has-icon" aria-label="Bold (Ctrl+B)"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" size="24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M14.7 11.3c1-.6 1.5-1.6 1.5-3 0-2.3-1.3-3.4-4-3.4H7v14h5.8c1.4 0 2.5-.3 3.3-1 .8-.7 1.2-1.7 1.2-2.9.1-1.9-.8-3.1-2.6-3.7zm-5.1-4h2.3c.6 0 1.1.1 1.4.4.3.3.5.7.5 1.2s-.2 1-.5 1.2c-.3.3-.8.4-1.4.4H9.6V7.3zm4.6 9c-.4.3-1 .4-1.7.4H9.6v-3.9h2.9c.7 0 1.3.2 1.7.5.4.3.6.8.6 1.5s-.2 1.2-.6 1.5z"></path></svg></button>
      <button type="button" class="components-button is-small has-icon" aria-label="Italic (Ctrl+I)"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" size="24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M12.5 5L10 19h1.9l2.5-14z"></path></svg></button>
       <button type="button" aria-pressed="false" class="components-button is-small has-icon" aria-label="Link"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" size="24" width="24" height="24" aria-hidden="true" focusable="false"><path d="M10 17.389H8.444A5.194 5.194 0 1 1 8.444 7H10v1.5H8.444a3.694 3.694 0 0 0 0 7.389H10v1.5ZM14 7h1.556a5.194 5.194 0 0 1 0 10.39H14v-1.5h1.556a3.694 3.694 0 0 0 0-7.39H14V7Zm-4.5 6h5v-1.5h-5V13Z"></path></svg></button>
   </div>
   <div contenteditable="true" class="edac-rich-textarea" style="min-height: 72px;" spellcheck="false"></div>
   <p class="edac-rich-textarea-help">Add a note explaining why this issue is being dismissed. Supports bold, italic, and links.</p>
</div>
```

Add `aria-labelledby` and `aria-describedby` to the contenteditable `div` referencing the label and helper text so that it's read out when entering the field.
